_targets.R

```diff
@@ -542,106 +542,114 @@ list(
-        source_maternity_extract,
-        source_mental_health_extract,
-        source_nrs_deaths_extract,
-        source_ooh_extract,
-        source_outpatients_extract,
-        source_prescribing_extract,
-        source_sc_care_home,
-        source_sc_home_care,
-        source_sc_sds,
-        source_sc_alarms_tele
-      )
-    ),
     tar_file_read(nsu_cohort, get_nsu_path(year), read_file(!!.x)),
     tar_target(
       homelessness_lookup,
       create_homelessness_lookup(
         year,
         homelessness_data = source_homelessness_extract %>% slfhelper::get_chi()
       )
-    ),
-    tar_target(
-      episode_file,
-      create_episode_file(
-        processed_data_list,
-        year,
-        homelessness_lookup = homelessness_lookup,
-        dd_data = source_dd_extract %>% slfhelper::get_chi(),
-        nsu_cohort = nsu_cohort %>% slfhelper::get_chi(),
-        ltc_data = source_ltc_lookup %>% slfhelper::get_chi(),
-        slf_pc_lookup = source_pc_lookup,
-        slf_gpprac_lookup = source_gp_lookup,
-        slf_deaths_lookup = slf_deaths_lookup %>% slfhelper::get_chi(),
-        sc_client = sc_client_lookup %>% slfhelper::get_chi(),
-        write_to_disk
-      )
-    ),
-    tar_target(
-      episode_file_tests,
-      process_tests_episode_file(
-        data = episode_file,
-        year = year
-      )
-    ) # ,
-    # tar_target(
-    #   cross_year_tests,
-    #   process_tests_cross_year(year = year)
-    # ), # ,
-    # tar_target(
-    #   individual_file,
-    #   create_individual_file(
-    #     episode_file = episode_file,
-    #     year = year,
-    #     homelessness_lookup = homelessness_lookup,
-    #     write_to_disk = write_to_disk
-    #   )
-    # ),
-    # tar_target(
-    #   individual_file_tests,
-    #   process_tests_individual_file(
-    #     data = individual_file,
-    #     year = year
-    #   )
-    # ) # ,
-    # tar_target(
-    #   episode_file_dataset,
-    #   arrow::write_dataset(
-    #     dataset = episode_file,
-    #     path = fs::path(
-    #       get_year_dir(year),
-    #       stringr::str_glue("source-episode-file-{year}")
-    #     ),
-    #     format = "parquet",
-    #     # Should correspond to the available slfhelper filters
-    #     partitioning = c("recid", "hscp2018"),
-    #     compression = "zstd",
-    #     version = "latest"
-    #   )
-    # ),
-    # tar_target(
-    #   individual_file_dataset,
-    #   arrow::write_dataset(
-    #     dataset = individual_file,
-    #     path = fs::path(
-    #       get_year_dir(year),
-    #       stringr::str_glue("source-individual-file-{year}")
-    #     ),
-    #     format = "parquet",
-    #     # Should correspond to the available slfhelper filters
-    #     partitioning = c("hscp2018"),
-    #     compression = "zstd",
-    #     version = "latest"
-    #   )
-    # )
+    )
+  ),
+  tar_target(
+    combined_deaths_lookup,
+    process_combined_deaths_lookup()
```

Comment:
I guess this needs to be moved out from targets.

Reply:
where does it go instead?

Reply:
Sorry for not making it perfectly clear. We should move it to `run_all_targets.R`. By moving it, we can make sure that combining deaths lookup is after the death file for each year being produced. I add a code suggestion below.